test/spec/filter.test.js

```diff
@@ -1,11 +1,13 @@
 /* global describe it */
 
-var _ = require('lodash-fp')
-var fixture = require('../fixture/filter-fixture')
-var filter = require('../../src/filter')
-var expect = require('chai').expect
+require('babel/register')
```

Comment:
I think cause tests/index.js has this you don't need it here? But I could be wrong